akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerTestSetupBase.scala

```diff
@@ -71,4 +71,13 @@ abstract class HttpServerTestSetupBase {
   def send(string: String): Unit = send(ByteString(string.stripMarginWithNewline("\r\n"), "UTF8"))
 
   def closeNetworkInput(): Unit = netIn.sendComplete()
+
+  def shutdownBlueprint(): Unit = {
+    netIn.sendComplete()
+    requests.expectComplete()
+
+    responses.sendComplete()
+    netOut.expectBytes(ByteString("HTT")) // ???
```

Comment:
Arh. Already figured it out, I have used it extensively in tests not consuming the reply, so this is of course the start of the reply... :P sorry for pinging you with such a stupid miss.